loadtesting/k6/readme.md

```diff
@@ -28,11 +33,27 @@ Install xk6 and build an executable file, tag is a neonlabsorg forked xk6-ethere
 ```
 
 Run load scenario:
+
+To run Send Neon scenario we need to prepare accounts and run load script:
+```bash
+./clickfile.py k6 prepare-accounts --network local --users 100 --balance 200
+
+./clickfile.py k6 run --network local --script ./loadtesting/k6/tests/sendNeon.test.js --users 100 --balance 200
+```
+
+To run Send ERC20 scenario we need to deploy the contract first and then run load script:
 ```bash
+./clickfile.py k6 deploy-erc20 --network local --balance 200 
+
 ./clickfile.py k6 run --network local --script ./loadtesting/k6/tests/sendNeon.test.js --users 100 --balance 200
 ```
 
-It is mandatory either set up `K6_USERS_NUMBER` and `K6_INITIAL_BALANCE` envs or pass `--users` and `--balance` flags to run the command. We need these values to prepare test accounts for further transfers transactions. 
+It is mandatory either set up `K6_USERS_NUMBER` and `K6_INITIAL_BALANCE` envs or pass `--users` and `--balance` flags to run the command. We need these values to set up options for k6 load scenario.
```

Comment:
As I understand you removed the possibility to set K6_USERS_NUMBER and K6_INITIAL_BALANCE
Only options is to pass `--users` and `--balance`

Reply:
fixed